storage/innobase/xtrabackup/doc/source/howtos/replication.setting-up.rst

```diff
@@ -0,0 +1,204 @@
+.. _pxb.replication.setting-up:
+
+================================================================================
+Setting up replication
+================================================================================
+
+:Overview: :ref:`pxb.replication`
+
+.. contents::
+   :local:
+
+
+.. _pxb.replication.setting-up/primary-server.backing-up:
+
+Make a backup on the primary server
+================================================================================
+
+Set up your primary server (replication source) as described in |mysql| documentation:
+
+- `Set a unique server_id
+  <https://dev.mysql.com/doc/refman/8.0/en/replication-howto-masterbaseconfig.html>`_
+- `Set up a replication user
+  <https://dev.mysql.com/doc/refman/8.0/en/replication-howto-repuser.html>`_ 
+
+Now, you are ready to make a backup of the data stored on the primary server.
+directory:
+
+.. code-block:: bash
+
+   $ xtrabackup --backup --target-dir=/data/backups/
+
+The destination directory |file.data.backups| now contains a directory the file
+:file:`xtrabackup_binlog_info`. This file stores both binary log
+coordinates and the |abbr.gtid| (provided the `GTID mode`_ is enabled).
+
+.. code-block:: bash
+
+   $ cat xtrabackup_binlog_info
+   mysql-bin.000002    1232        c777888a-b6df-11e2-a604-080027635ef5:1-4
+
+This information is also printed to standard output after the backup is made.
+
+
+.. _pxb.replication.setting-up/backup.preparing:
+
```

Comment:
In Full backup - it's only Preparing a backup - the titles should be the same - this title is more clear
Preparing the backup for restoration

Reply:
Updated to make it more specific: Preparing the backup for restoring on a replica. In the context of full backup we are talking about a local server. I believe the term prepare is not accurate per se. Its job is synchronizing the backup with the current state of the source of backup. This short form prevails because you actually run xtrabackup --prepare